src/components/ResponsiveImage.astro

```diff
@@ -20,6 +20,7 @@ export type ResponsiveImageItem = {
     src: GetImageResult;
   }[];
   sizes?: string;
+  link?: string;
```

Comment:
Put this in the logoList type definition instead of here, this component could be used somewhere else and putting link here isn't a good idea.

Reply:
Yes!